TrafficCapture/trafficReplayer/src/main/java/org/opensearch/migrations/replay/TrafficReplayer.java

```diff
@@ -604,15 +618,16 @@ private static SourceTargetCaptureTuple getSourceTargetCaptureTuple(RequestRespo
         }
     }
 
-    public void runReplay(ITrafficCaptureSource trafficChunkStream,
-                          CapturedTrafficToHttpTransactionAccumulator trafficToHttpTransactionAccumulator) {
+    public void pullReplayFromSourceToAccumulator(
```

Comment:
nit: Stopped me a bit to understand what it meant to `pullReplay` would prefer `pullRecords` but don't know if that is much better

Reply:
I'm nervous about using the word Record since that's now a Kafka term.  Same with TrafficStream... However, I think that this should be pullCaptureFromSourceIntoAccumulator - or something more like that.  This isn't a replay yet, but it is a capture.

Reply:
Agree capture is better